go/proto/hydra/v1/routing.proto

```
@@ -0,0 +1,64 @@
+syntax = "proto3";
+
+package hydra.v1;
+
+import "dev/restate/sdk/go.proto";
```

Buf (1.57.2)

import "dev/restate/sdk/go.proto": file does not exist

(COMPILE)